templates/zerver/help/search-for-messages.md

```diff
@@ -54,7 +54,7 @@ shared history](/help/stream-permissions), including messages sent
 before you joined the stream (or organization) or those sent to public
 streams you are not subscribed to.
 
-By default, Zulip searches messages in  your history, i.e. the
+By default, Zulip searches messages in your history, i.e. the
```

Comment:
This should not be a separate commit. You should read up on git, especially thing like `git reset`, `git commit --amend`, etc. You will have to uncommit this change locally, and then use `--amend` to add it to the 1st commit.

You can find details on how to do this on a lot of websites.

Reply:
@stableapple Thanks for these changes!

I agree with @vinitS101 that this should all just be a single commit.  You can do `git rebase -i master` and put "f" next to the second commit to accomplish this.  I also agree with Vinit's advice on reading up on git.